contrib/devtools/security-check.py

```diff
@@ -184,6 +198,14 @@ def check_PE_NX(executable) -> bool:
     bits = get_PE_dll_characteristics(executable)
     return (bits & IMAGE_DLL_CHARACTERISTICS_NX_COMPAT) == IMAGE_DLL_CHARACTERISTICS_NX_COMPAT
 
+def check_PE_control_flow_instrumentation(executable) -> bool:
+    stdout = run_command([OBJDUMP_CMD, '-d', '--section=.text', executable])
+
+    for line in stdout.splitlines():
+        if 'endbr64' in line:
```

Comment:
It's too bad that x86 has variable-length instructions (so it's tricky to find instruction binaries), or we could just look in the section contents for the raw instruction instead of parsing text.

Reply:
You've implemented this!